Makefile

```diff
@@ -1,4 +1,6 @@
 .PHONY: build
 
 build:
-	docker build -t twistlock/cloud-discovery .
\ No newline at end of file
+	rm -rf licenses; mkdir -p licenses; find . | grep LICENSE | while IFS= read -r pathname; do filename=$$(echo $$pathname | tr "\.\/" "_" | sed "s/^__//"); cp $$pathname licenses/$$filename; done
```

Comment:
I'm not sure about that this complexity is needed; can't you hard-copy a license file.

Reply:
Some marketplaces require ALL of the licenses (about 30) to be included in the image. I could also make a different Makefile job like `build-with-licenses` for that case.